src/content/1.7/development/internationalization/make-a-new-language-available-to-be-used-in-the-back-office.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Make a new language available to be used in the Back Office
+aliases:
+  - /1.7/development/make_a_new_language_available_to_be_used_in_the_back_office
+  - /1.7/development/make-a-new-language-available-to-be-used-in-the-back-office
+---
+
+# Make a new language available to be used in the Back Office
```

Comment:
_Activate a language in PrestaShop_

Reply:
@LouiseBonnard In this case, I change the URL to the page to be logic.

Reply:
Done